### PR TITLE
fix(background/Heartbeat): use `alarm` instead of `storage.set`

### PR DESCRIPTION
**Pull request by sidvishnoi**
## Context

Closes https://github.com/interledger/web-monetization-extension/issues/492
Pair programmed with @raducristianpopa and @sidvishnoi

### CODE REVIEW

**Comment by raducristianpopa** [comment]
<h2>Extension builds preview</h2>

<table role="table">
<thead>
  <tr>
    <th>Name</th>
    <th>Link</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td align="center">Latest commit</td>
    <td align="center">3253f4ba5564c6f41ccc9a44e0fc19fbe1d43a7b</td>
  </tr>
  <tr>
    <td align="center">Latest job logs</td>
    <td align="center"><a href="https://github.com/interledger/web-monetization-extension/actions/runs/10298602019">Run #10298602019</a></td>
  </tr>
  <tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/26941539195/artifacts/1789377698">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/26941539195/artifacts/1789377701">Download</a></td></tr>
</tbody>
</table>

File: src/background/services/heartbeat.ts [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
+    setTimeout(
+      () => alarms.create('keep-alive-alarm-3', { periodInMinutes: 1 }),
+      45 * 1000
+    )
 
-  run() {
-    void this.storage.get(['version'])
+    alarms.onAlarm.addListener(() => {
+      // doing nothing is enough to keep it alive
+    })
```
Comment (sidvishnoi) [inline src/background/services/heartbeat.ts:34]: I wonder if we can create a single alarm with `when: now + 15s`, and recreate it when it fires?

Comment (sidvishnoi) [reply src/background/services/heartbeat.ts:34]: Not possible: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/alarms/create#alarminfo

**Review by raducristianpopa — APPROVED** [review]
Done during pair-programming.

**Pull request merged**